### PR TITLE
Updated Dio version to 5.0.0

### PR DESCRIPTION
**Pull request by MatyasK**

### CODE REVIEW

**Comment by kuhnroyal** [comment]
@felangel Can you please approve the workflow and possibly release this?

File: packages/fresh_dio/pubspec.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
```
Comment (kuhnroyal) [inline packages/fresh_dio/pubspec.yaml:10]:
```suggestion
  sdk: ">=2.15.0 <3.0.0"
```

That is the min Dio requirement

**Comment by SebastianKlaiber** [comment]
@MatyasK I think you need to adjust the [test](https://github.com/felangel/fresh/blob/master/packages/fresh_dio/test/fresh_test.dart#L260) as well. Because `sendTimeout` and `receiveTimout` changed in Dio 5 from int to Duration.

image

**Comment by MatyasK** [comment]
fixed failing tests

**Comment by kuhnroyal** [comment]
Please review & release :)

**Review by felangel — DISMISSED** [review]
LGTM ✅

**Comment by felangel** [comment]
Looks like there are some analysis warnings that need to be fixed but otherwise it looks great! Let me know if you need any help and thanks again for the PR 👍

**Comment by MatyasK** [comment]
@felangel analyzer warnings are fixed

**Review by felangel — DISMISSED** [review]

**Comment by felangel** [comment]
> @felangel analyzer warnings are fixed

Thanks looks like you also need to run `dart format` 👍

**Comment by felangel** [comment]
This has been addressed by #79

**Pull request closed without merging**